backend/src/helpers/uuid.js

```diff
@@ -0,0 +1,7 @@
+// copy paste from https://stackoverflow.com/a/2117523
+export default function uuidv4() {
```

Comment:
Just use uuid library, no need here to re-invent the wheel :smile:

Reply:
ok :(